scripts/scenario_07_quick_basic.lua

```diff
@@ -347,14 +352,18 @@ function update(delta)
         -- Note that players can win even if they destroy the enemies by blowing themselves up.
         if enemy_count == 0 then
             victory("Human Navy")
-            setBanner("Mission: SUCCESS")
+            local text = string.format("Mission: SUCCESS (%d seconds left)", math.floor(gametimeleft))
```

Comment:
Does this text fit in the global message box? I remember the font being quite large and the box not being super large.

Reply:
Probably not, but this should not be a problem. Compare the detailed mission result in Borderline Fever.

Is there another possibility for a longer interesting message?

If it is a problem, we can remove this detail. The other changes (and the upcoming) are more relevant.